frontend_lib/src/component/DropdownMenu/DropdownMenu.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import DropdownMenuItem from './DropdownMenuItem.jsx'
+
+const DropdownMenu = props => {
+  return (
+    <div className='dropdown'>
+      <button
+        aria-expanded='false'
+        aria-haspopup='true'
+        className={classnames(
+          'btn dropdown-toggle',
+          props.isButton ? 'nohover primaryColorBorder' : 'transparentButton',
+          props.customClassButton
```

Comment:
`customClassButton`?

Reply:
as it is a generic component, the calling component may want to apply some specific property (for example, a background color) to the button that toggles the drop-down menu...

Reply:
after reading the other comments I understood that you might be asking about the name. I update to buttonCustomClass :smile: